versions/wmde10.env

```diff
@@ -10,9 +10,9 @@ WDQS_FRONTEND_COMMIT_HASH=fb1e5bfc12491a41e45389a613816967c302dd43
 QUICKSTATEMENTS_COMMIT_HASH=a2d155f5f90ca7ba26ebd6ce7f2892de9999961a
 # Head of master on 03/01/2023
 MAGNUSTOOLS_COMMIT_HASH=68c34ee04d3c6d68fb9e2106821f9686a96d2075
-# 1.0.2 latest release on 03/01/2023
+# 1.0.2 latest release on 03/01/2023 (doesn't state 1.38 compatability)
```

Comment:
so does it work properly?

Reply:
The tests pass, that is all I check as part of the creation of this PR
There are also product tests that will happen once this PR is merged.
This comment is only to help devs in the future if they glance at the commit hash and want to know what that actually means